### PR TITLE
Android client

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: clients/Android/ultrablue/app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="fr.gouv.ssi.ultrablue">
+
+    <application
```
Comment (kerneis-anssi) [inline clients/Android/ultrablue/app/src/main/AndroidManifest.xml:5]: What is the minimum SDK version?

https://developer.android.com/guide/topics/manifest/uses-sdk-element

**Review by kerneis-anssi — CHANGES_REQUESTED** [review]

File: clients/Android/ultrablue/app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (kerneis-anssi) [reply clients/Android/ultrablue/app/src/main/AndroidManifest.xml:5]: Let's address this in a follow-up commit.

File: clients/Android/ultrablue/app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -14,4 +16,11 @@
     <string name="delete_button_desc">Delete button</string>
     <string name="address_title">Address:</string>
 
+    <string name="rename_device_dialog_title">Choose new device name</string>
+    <string name="delete_device_dialog_title">test %s</string>
```
Comment (kerneis-anssi) [inline clients/Android/ultrablue/app/src/main/res/values/strings.xml]: This should be `Delete`, not `test %s`.

**Review by kerneis-anssi — CHANGES_REQUESTED** [review]

File: clients/Android/ultrablue/app/src/main/res/values/strings.xml (hunk 2 above)
Comment (ghost) [reply clients/Android/ultrablue/app/src/main/res/values/strings.xml]: Fixed.

**Review by kerneis-anssi — APPROVED** [review]

**Pull request merged**